spyder/plugins/variableexplorer/widgets/arrayeditor.py

```diff
@@ -764,42 +776,51 @@ def setup_and_check(self, data, title='', readonly=False,
         self.layout.addWidget(self.stack, 1, 0)
 
         # ---- Toolbar and actions
-        toolbar = SpyderToolbar(parent=None, title='Editor toolbar')
+        toolbar = SpyderToolbar(parent=self, title='Editor toolbar')
         toolbar.setStyleSheet(str(PANES_TOOLBAR_STYLESHEET))
 
-        self.copy_action = create_action(
-            self, _('Copy'), icon=ima.icon('editcopy'),
+        self.copy_action = self.create_action(
+            ArrayEditorActions.Copy,
+            text=_('Copy'),
+            icon=ima.icon('editcopy'),
```

Comment:
```suggestion
            icon=self.create_icon('editcopy'),
```

`SpyderWidgetMixin` also has a high level API to assign icons to actions, so we can use it here too.